docs/getting_started/basics/batch.md

```diff
@@ -0,0 +1,35 @@
+# Batch Execution
+
+Shift is able to run workflows without the need of a UI interface. This is called Batch mode and can be used via the Shift Python API.
+
+## Opening a Workflow
+
+The following snippet of code can be used to open a Shift workflow file:
+
+```python
+from shift.core import files
+
+workflow, catalog_manifest = files.openBatchWorkflow("<path_to_your_workflow_file>")
+```
+
+The execution of this method will return the `SWorkflow` Python class containing the workflow and a Python dictionary containing the information on the Python modules loaded by the workflow. Have a look at the [Shift API](../../reference/api.md) for more information on the method and the `SWorkflow` class.
+
+## Executing a Workflow
+
+Once the workflow file is opened, its execution can be triggered with the following code:
+
+```python
+result = workflow.execute()
+```
+
+External inputs for the workflow can be set by providing a Python dictionary as first argument of the `workflow.execute` call. This dictionary should be formatted using the input plug names from the `Input` operator as keys and the content that should be passed as values.
```

Comment:
I would suggest to add another example with parameters. Maybe:
```suggestion
External inputs for the workflow can be set by providing a Python dictionary as first argument of the `workflow.execute` call. This dictionary should be formatted using the input plug names from the `Input` operator as keys and the content that should be passed as values.

```python
workflowExternals = {
    "myBooleanPlug": True,
    "myStringPlug": "Hello world"
}
result = workflow.execute(workflowExternals)
```
```

Reply:
done!